### ISSUE TITLE
Tip, Tip Group and Tip Manager - Provide a valid label for form fields - (2036570369)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5612]
### Module: 
 36 Tip, Tip Group and Tip Manager 

### Violation: 
 Provide a valid label for form fields 

### WCAG Reference: 
- [1.3.1 Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [3.3.2 Labels or Instructions](https://www.w3.org/WAI/WCAG21/Understanding/labels-or-instructions.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570369 

### Severity: 
 10 

### Description: 
 [Issue]
There are "close" buttons that has identical accessible names.

[User Impact]
Screen reader users will be unable to determine the purpose of these buttons. Speech input users will have difficulty activating them.

[Code Reference] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Close" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Close</div>
</button>
 ``` 

### Note: 
 [Suggestion]
Provide a valid label for form fields. This includes providing accessible names for buttons. The accessible name of a button can be set with an aria-label attribute. Good accessible names are both concise and descriptive.

[Compliant Code Example] 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Close Test 1. Tip" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Close</div>
</button>
 ```
 

### Media Type: 
 Forms 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
This is a result of the testing mechanism and is not a valid bug. Will add an a11y sample to the component doc page for folks. Closing this issue out.
